app/src/main/java/io/intrepid/contest/models/InvitationResponse.java

```diff
@@ -0,0 +1,17 @@
+package io.intrepid.contest.models;
+
+import com.google.gson.annotations.Expose;
+
+import java.util.Date;
+import java.util.UUID;
+
+public class InvitationResponse {
+    @Expose
+    public UUID id;
+    @Expose
+    public UUID contestantId;
```

Comment:
Yeah I don't really like the idea of removing it and adding it back in if we will 💯  need it.